New package: ShortFFTs v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: ShortFFTs
- Repository: https://github.com/eschnett/ShortFFTs.jl
- Created by: @eschnett
- Version: v1.0.0
- Commit: 711c8786859a773a18777ebd4dc0c1acdc8dd14c
- Git reference: HEAD
- Description: Efficient short Fast Fourier Transforms

goerz commented:
Thank you for submitting your package! However, please make sure to add some documentation before registering. [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a description of the package's purpose and a small usage example in the README

JuliaHubRegistrator committed:
New package: ShortFFTs v1.0.0

UUID: 3a4051df-11c6-4922-af5a-d7072afac5a2
Repo: https://github.com/eschnett/ShortFFTs.jl.git
Tree: 226a80838f99e0b9c92ad2b322141f5a8ded6a0f

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8
